src/sage/algebras/letterplace/free_algebra_element_letterplace.pyx

```diff
@@ -139,7 +139,7 @@ cdef class FreeAlgebraElement_letterplace(AlgebraElement):
             sage: sorted(p)   # indirect doctest
             [((0, 0, 0, 1, 0, 0, 0, 1), 2), ((0, 1, 0, 0, 0, 0, 1, 0), 1)]
         """
-        cdef dict d = self._poly.dict()
+        cdef dict d = self._poly.monomial_coefficients()
         yield from d.iteritems()
```

Comment:
should I do
```suggestion
        yield from d.items()
```
throughout?

Reply:
This can be changed since it was part of the Python3 transition.